packages/salesforcedx-apex-debugger/src/adapter/apexDebug.ts

```diff
@@ -740,6 +803,7 @@ export class ApexDebug extends LoggingDebugSession {
         response.message = error;
       }
     }
+    this.resetIdleTimer();
```

Comment:
Reset idle timer when user selects "Run". This is needed in case they run to the end of a debugged request. There's no request for variables, but we still need to reset the timer because they technically did something with the debugger.